std/concurrency.d

```diff
@@ -842,80 +795,69 @@ body
  * $(REF Duration, core,time) has passed. It returns $(D true) if it received a
  * message and $(D false) if it timed out waiting for one.
  */
-bool receiveTimeout(T...)( Duration duration, T ops )
+bool receiveTimeout(T...)(Duration duration, T ops)
 in
 {
     assert(thisInfo.ident.mbox !is null,
-           "Cannot receive a message until a thread was spawned "
-           ~ "or thisTid was passed to a running thread.");
+        "Cannot receive a message until a thread was spawned or thisTid was passed to a running thread.");
 }
 body
 {
-    checkops( ops );
+    checkops(ops);
 
-    return thisInfo.ident.mbox.get( duration, ops );
+    return thisInfo.ident.mbox.get(duration, ops);
 }
 
 @safe unittest
 {
-    static assert( __traits( compiles,
-                      {
-                          receiveTimeout( msecs(0), (Variant x) {} );
-                          receiveTimeout( msecs(0), (int x) {}, (Variant x) {} );
-                      } ) );
+    static assert(__traits(compiles, {
+        receiveTimeout(msecs(0), (Variant x) {  });
+        receiveTimeout(msecs(0), (int x) {  }, (Variant x) {  });
+    }));
 
-    static assert( !__traits( compiles,
-                       {
-                           receiveTimeout( msecs(0), (Variant x) {}, (int x) {} );
-                       } ) );
+    static assert(!__traits(compiles, {
+        receiveTimeout(msecs(0), (Variant x) {  }, (int x) {  });
+    }));
 
-    static assert( !__traits( compiles,
-                       {
-                           receiveTimeout( msecs(0), (int x) {}, (int x) {} );
-                       } ) );
+    static assert(!__traits(compiles, {
+        receiveTimeout(msecs(0), (int x) {  }, (int x) {  });
+    }));
 
-    static assert( __traits( compiles,
-                      {
-                          receiveTimeout( msecs(10), (int x) {}, (Variant x) {} );
-                      } ) );
+    static assert(__traits(compiles, {
+        receiveTimeout(msecs(10), (int x) {  }, (Variant x) {  });
```

Comment:
Is there any reason why you convert `{}` to `{ }`?
AFIAK `{}` is a very common style in Phobos (485 occurrences).